tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -136,7 +136,7 @@ public static string GetNativeLoaderPath()
                 ("win", "X64")     => "Datadog.AutoInstrumentation.NativeLoader.x64.dll",
                 ("win", "X86")     => "Datadog.AutoInstrumentation.NativeLoader.x86.dll",
                 ("linux", "X64")   => "Datadog.Trace.ClrProfiler.Native.so",
-                ("linux", "Arm64") => "Datadog.AutoInstrumentation.NativeLoader.so",
+                ("linux", "Arm64") => "Datadog.Trace.ClrProfiler.Native.so",
```

Comment:
nit: You can combine both of the `linux` conditions into a single pattern

```suggestion
                ("linux", "X64" or "Arm64")   => "Datadog.Trace.ClrProfiler.Native.so",
```